app/src/main/java/com/example/superheroapi/data/SuperHeroRepository.kt

```diff
@@ -5,19 +5,13 @@ import com.example.superheroapi.model.SuperHeroSearch
 import com.example.superheroapi.network.SuperHeroApiService
 
 interface SuperHeroRepository {
-    suspend fun searchHero(apiKey: String, name: String): SuperHeroSearch
     suspend fun getHero(apiKey: String, id: String): SuperHero
 }
 
 class SuperHeroNetworkRepo(
     private val superHeroApiService: SuperHeroApiService
 ) : SuperHeroRepository {
-    override suspend fun searchHero(apiKey: String, name: String): SuperHeroSearch {
-        return superHeroApiService.searchHero(apiKey, name)
-    }
-
```

Comment:
Removed the "searchHero" function. To keep this app simple, we assume the user knows the search bar can ONLY accept a superhero id. We can possibly come back to this once we officially learn to use the Compose Navigation library to properly deal with multiple screens.